sha2/build.rs

```diff
@@ -6,24 +6,35 @@ fn main() {
 
     let mut build256 = cc::Build::new();
     let (sha256_path, sha512_path) = if target_arch == "x86" {
-        ("src/sha256_x86.S", "src/sha512_x86.S")
+        (["src/sha256_x86.S"].to_vec(), ["src/sha512_x86.S"].to_vec())
```

Comment:
I understand it's just a build script, but unnecessary allocations irk me a bit. :) I think using `&'static [&'static str]` type for `sha256_path` and `sha512_path` should work without issues.